frontend/src/pages/ContributionView.jsx

```
@@ -0,0 +1,286 @@
+      } catch (error) {
+        console.error("Error fetching course data:", error);
+        setLoading(false);
+      }
+    };
+
+    fetchCourseData();
+  }, []);  // Removed id dependency since we're using fixed example
+
+  // Set current video when course data or active index changes
+  useEffect(() => {
+    if (contentData[exampleId] && contentData[exampleId].content && contentData[exampleId].content.length > 0) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)